Firestore/core/src/firebase/firestore/util/CMakeLists.txt

```diff
@@ -210,6 +210,7 @@ cc_library(
     status_win.cc
```

Comment:
You haven't added `vector_of_ptr.h` here, is this intentional?

Reply:
Ha! No. I noticed that all these were missing when going to add it and then forgot to actually do so!